docs/conf.py

```
@@ -71,7 +72,7 @@ def setup(sphinx):
 #
 # The short X.Y version.
 with open('../CMakeLists.txt', 'r', encoding='utf8') as f:
-    version = re.search('PROJECT_VERSION "([^"]+)"', f.read()).group(1)
+    version = re.search('PROJECT_VERSION "([^"]+)"', f.read()).group(1) # type: ignore
```

If we suppress errors like this, most probably they won't be fixed anytime soon. So maybe it would be better to use a proper type right away: `version: Optional[re.Match] = re.search(....)`. Or, we can create a follow-up ticket to address them, which I guess is not that bad either.